_post/spaceranger-23/2020-09-23-Functional Turtle.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: Post
+author: spaceranger-23
+title: spaceranger-23 Functional Turtle
```

Comment:
It is safest to enclose the title in `"`s to make it a string, or you could enter invalid characters.